modules/pam_faillock/main.c

```diff
@@ -201,8 +197,9 @@ do_allusers(struct options *opts)
 	for (i = 0; i < rv; i++) {
 		if (userlist[i]->d_name[0] == '.') {
 			if ((userlist[i]->d_name[1] == '.' && userlist[i]->d_name[2] == '\0') ||
-			    userlist[i]->d_name[1] == '\0')
-				continue;
+				userlist[i]->d_name[1] == '\0') {
+					continue;
```

Comment:
I do not really see this as an improvement.

Reply:
convert the spaces to tab, also add pair {} to continue statement.